lib/alexa.js

```diff
@@ -367,6 +367,9 @@ function ResponseBuilder(self) {
 
                 responseObject.response.directives = [audioPlayerDirective];
                 return this;
+            },
+            'setResponse': function () {
```

Comment:
You don't need to setResponse when responseObject is already referencing self.response. You can just do:
```javascript
this.response.speak('foo').listen('bar');
this.emit(':responseReady');
```
And you will get the response!